cmd/describe/cluster/cmd.go

```diff
@@ -413,12 +414,103 @@ func run(cmd *cobra.Command, argv []string) {
 			" - Details:                 %s\n",
 			str, reason.Summary(), reason.Details())
 	}
+
+	inflightChecks, err := r.OCMClient.GetInflightChecks(cluster.ID())
+	if err != nil {
+		r.Reporter.Errorf("Failed to get inflight checks for cluster '%s': %v", cluster.ID(), err)
+		os.Exit(1)
+	}
+	if len(inflightChecks) > 0 {
+		summaries := []string{}
+		for _, inflight := range inflightChecks {
+			if inflight.State() == "failed" {
+				summary := fmt.Sprintf("\t"+
+					"ID:                 %s\n"+
+					"\tLast run:           %s\n",
+					inflight.ID(), inflight.EndedAt().Format("Jan _2 2006 15:04:05 MST"))
+				details, err := parseInflightCheckDetails(inflight)
+				if err != nil {
+					r.Logger.Errorf("Unexpected error parsing inflight details '%s: %v", inflight.ID(), err)
+					continue
+				}
+				summary += details
+				summaries = append(summaries, summary)
+			}
+		}
+		if len(summaries) > 0 {
+			str += fmt.Sprintf("Failed Inflight Checks:\n%s\n", strings.Join(summaries, "\n"))
+			str += fmt.Sprintf("\tPlease run `rosa verify network -c %s` after adjusting"+
+				" the cluster's network configuration to remove the warning related to"+
+				" 'Egress URL access' if present", cluster.ID())
+		}
+	}
+
 	str = fmt.Sprintf("%s\n", str)
 
 	// Print short cluster description:
 	fmt.Print(str)
 }
 
+var mapInflightErrorTypeToTitle = map[string]string{
+	"egress_url_errors": "Egress URL access",
```

Comment:
Egress URL access -> Egress URL errors?

Reply:
So we call it egress errors within, I wanted something like connectivity but I find it too long. Access felt fairly ok

Reply:
Added 'access issues'